libs/core/synchronization/include/hpx/synchronization/async_rw_mutex.hpp

```diff
@@ -177,12 +177,17 @@ namespace hpx { namespace experimental {
             async_rw_mutex_access_wrapper& operator=(
                 async_rw_mutex_access_wrapper const&) = default;
 
-            operator ReadT&()
+            ReadT& get()
```

Comment:
Is the caller allowed to change the value stored in the object?

Reply:
No, `ReadT` will always be const (it's enforced by the `async_rw_mutex`, not the `async_rw_mutex_access_wrapper`).

Reply:
Would changing it to  `ReadT& get() const` be an option in this case?

Reply:
Yes, good point!

Reply:
Done.